test/kubemark/pre-existing/README.md

```diff
@@ -0,0 +1,45 @@
+# Kubemark Pre-existing Provider Guide
+
+**Pre-existing Kubemark Master**
```

Comment:
>The kubemark setup that we are using is:
    1. some kubernetes cluster
    2. a separate VM where kubemark master is running
    3. hollow-nodes that are running on cluster from 1)
>

The target of this patch is to use the same model with (2) setup by the developer instead of the provider scripts. I wasn't clear in the doc because I was using the wrong terminology. I considered the kubemark cluster to be an entire cluster with a master and minions. But now I get that it's meant to be a single master.